### DISCUSSION TITLE
A question to the creator!

### DISCUSSION THREAD

**Discussion by ShadoW2551** [xCollateral/VulkanMod#275] [Q&A]
Hi
Can you tell me how you ported opengl to vulkan? Like you, I'd like to port another game's render from opengl to vulkan :)

**Comment by sqrvrt** [comment]
future note: please don't name a thread like that

**Comment by ElectricLizzzard** [comment]
You can't port OpenGL to Vulkan. You just write a Vulkan renderer. That's how it works in engines and in that case too. What VulkanMod is doing is changing vanilla OpenGL renderer and put the custom Vulkan renderer in use. Vulkan is NOT a drop-in replacement for OpenGL, this is an entire new API.

Sodium is working that way too, but it changes the vanilla OpenGL renderer to custom well coded and optimized OpenGL renderer.
